packages/files-ui/cypress/fixtures/loginData.ts

```diff
@@ -1,4 +1,4 @@
 export const testPrivateKey = "0x4fd9a44c051834e28a919096f154e3f805f58d8f2d90a1ab9a29b0b46a6ac363"
 export const testAddress = "0xDa1f70D176E0F7F16f2a3bB509cD37Bd36aFD902"
 export const testAccountPassword = "correct horse"
-export const localHost = "http://localhost:3000"
\ No newline at end of file
+export const localHost = "https://files-ui-stage-pr-1927.onrender.com"
```

Comment:
let's put this back :) 

```suggestion
export const localHost = "http://localhost:3000"
```